src/main/java/de/torui/coflsky/gui/CoflGui.java

```diff
@@ -0,0 +1,471 @@
+                    .setX(new PixelConstraint(4f))
+                    .setTextScale(new PixelConstraint((float) fontScale*2f));
+    
+                new UIWrappedText(uppercaseFirstLetter(description)).setChildOf(exampleFeature)
+                    .setX(new PixelConstraint(4f))
+                    .setWidth(new PixelConstraint(350))
+                    .setColor(new Color(187,187,187))
+                    .setY(new PixelConstraint(23f*(float) fontScale))
+                    .setTextScale(new PixelConstraint((float) fontScale*1f));
+            
+                
+                String type = setting.get("type").getAsString();
+                if(type.equals("Boolean")) {
+                    UIComponent comp = new SwitchComponent(setting.get("value").getAsBoolean()).setChildOf(exampleFeature);
+                    ((SwitchComponent) comp).onValueChange((value)->{
+                        if(value!=setting.get("value")) {
+                            updateSetting(setting.get("key").getAsString(),value);
+                        }
+                        return Unit.INSTANCE;
+                    });
+                }
+
+                if(type.equals("String")) {
+                    if(setting.get("value")==null) continue;
+                    String formattedValue = "";
+                    try {formattedValue = setting.get("value").getAsString();} catch (Exception e) {continue;}
+                    formattedValue = formattedValue.replaceAll("\\\"","");
+                    formattedValue = formattedValue.replaceAll("§", "&");
+                    UIComponent comp = new TextComponent(formattedValue, "", false, false).setChildOf(exampleFeature);
+                    ((TextComponent) comp).onValueChange((value)->{
+                        if(!value.toString().equals(setting.get("value").toString()) && value.toString().length()>0) {
+                            value = ((String) value).replaceAll("&", "§");
+                            updateSetting(setting.get("key").getAsString(),value);
+                        }
+                        return Unit.INSTANCE;
+                    });
+                }
+    
+                if(type.equals("Int64") || type.equals("Double") || type.equals("Int32")) {
+                    UIComponent comp = new TextComponent(setting.get("value")+"", "", false, false).setChildOf(exampleFeature);
+                    ((TextComponent) comp).onValueChange((value)->{
+                        value=value.toString().replaceAll("[^0-9m.btk]","");
+                        if(!value.toString().equals(setting.get("value").toString()) && value.toString().length()>0) {
+                            System.out.println("DIFFERENCE '"+value+"' '"+setting.get("value")+"'");
+                            updateSetting(setting.get("key").getAsString(),value);
+                        }
+                        return Unit.INSTANCE;
+                    });
+                }
+                index++;
+            }
```

Comment:
could you split this into multiple functions to fit on one screen each?

Reply:
sure